torch_xla/distributed/parallel_loader.py

```diff
@@ -74,6 +77,7 @@ def __init__(self,
     self._devices = [torch.device(x) for x in devices]
     self._batchdim = batchdim
     self._fixed_batch_size = fixed_batch_size
+    self._per_device_samples = len(loader) // len(devices)
```

Comment:
Does already DistributedSampler take into account world size ?

Reply:
Better be 😄 ... otherwise nothing will work